client/my-sites/media-library/upload-button.jsx

```diff
@@ -73,6 +81,7 @@ module.exports = React.createClass( {
 						accept={ this.getInputAccept() }
 						multiple
 						onChange={ this.uploadFiles }
+						onClick={ this.onClick }
```

Comment:
We should rename this method according to what the method does. One line above there is a nice example. When the  `onChange` event happens, then `uploadFiles`.
`event` => `action to take`.
Could it be something like `openMediaModal`?